resources/views/errors/401.blade.php

```diff
@@ -0,0 +1,26 @@
+@extends('layouts/basic')
```

Comment:
Do we actually need this? Seems to be a duplicate of our 403 (Forbidden) status page.